### ISSUE TITLE
Clarification on sensory perception of pain

### ISSUE THREAD

**Issue by wdduncan** [geneontology/go-ontology#28236]
The current definition for [sensory perception of pain](http://purl.obolibrary.org/obo/GO_0019233) is:  
> The series of events required for an organism to receive a painful stimulus, convert it to a molecular signal, and recognize and characterize the signal. Pain is medically defined as the physical sensation of discomfort or distress caused by injury or illness, so can hence be described as a harmful stimulus which signals current (or impending) tissue damage. Pain may come from extremes of temperature, mechanical damage, electricity or from noxious chemical substances. This is a neurological process.

Is this definition meant to include the subjective experience/feeling of pain? That is, does a process of `sensory perception of pain` entail that the organism feels pain?

Some of the cases that we are needing to represent may involve the activation of pain pathways, but the organism may not feel pain. For example, if a patient is in a coma or under general anesthesia, the nervous system may carry pain signals, but the patient doesn't feel the pain. In such a case, is `sensory perception of pain` involved?

Another case to consider is the experience of neuropathic pain which results from damage to nerve tissue. Does the 'sensory' part of 'sensory perception of pain' include cases such as this too?

**Comment by cmungall** [comment]
These are good questions!

A few things to bear in mind:

* GO processes represent the execution of an evolved genetically-encoded biological module or program, and shouldn't be used for things like neuropathic pain
* GO doesn't cover subjective experience

The sensory perception branch doesn't conform to our new DPs yet but I think it should be organized as follows:

* sensory perception of X (aka X nociception)
    * [p] detection of stimulus of X
    * [p] downstream processing of X

We also have response to X terms but these are not linked to the above hierarchy. But the causal flow is `detection of stimulus --> processing --> response`

Because GO focuses on genes we haven't instantiated the internal processing steps, which might be useful for annotating neurons, but @dosumis's perspective would be good here.

I think we need to clarify the starts and ends of everything as right now annotations are a bit inconsistent, eg many genes are annotated to both the detection of stimulus and response

**Comment by wdduncan** [comment]
> GO processes represent the execution of an evolved genetically-encoded biological module or program, and shouldn't be used for things like neuropathic pain
GO doesn't cover subjective experience

Thanks for the clarification! This definitely informs us about directions we should go with the pain ontology. We must represent neuropathic and noplastic pain, and the subjective experience of pain (and its related measurements) need to be accounted for too.

We may still be able to use the `sensory perception of pain` term for the firing of nociceptors. What do you think @addiehl ?

**Comment by dosumis** [comment]
For firing of nociceptors you should use 'detection of stimulus involved in sensory perception of pain' (this is the pattern we use for sensory neurons in general).

<img width="527" alt="Image" src="https://github.com/user-attachments/assets/e0b9f328-f19c-454e-9870-64f4166ad1dd" />

We use the broader "sensory perception" terms to annotate downstream neurons processing a sensory stimulus.   It would be hard to put any limits on how far downstream this should be. 

I'll defer to Chris on the limits of GO - but I suspect the biology is not clear cut and that many of the same downstream circuits will be involved sensory perception of pain a neuropathic pain.  It wouldn't surprise me if there were a fuzzy boundary between sensed and neuropathic pain due to differences in pain thresholds.

**Comment by cmungall** [comment]
I agree that it seems likely the same circuits are involved, so using GO to annotate those seems fine.

**Comment by cmungall** [comment]
I'm going to close this now as I think the main question has been answered. I think we could do with better starts/ends definitions for perception/detection-of-stimulus/response but this should be a separate ticket

**Comment by wdduncan** [comment]
If the GO isn't about feeling pain, it would be good to remove mention of `physical sensation of discomfort or distress` from the definition. I.e.:   
> Pain is medically defined as the physical sensation of discomfort or distress caused by injury or illness, so can hence be described as a harmful stimulus which signals current (or impending) tissue damage. Pain may come from extremes of temperature, mechanical damage, electricity or from noxious chemical substances. This is a neurological process.
